crates/sui-tool/src/commands.rs

```diff
@@ -273,11 +268,44 @@ pub enum ToolCommand {
         /// value based on number of available logical cores.
         #[clap(long = "num-parallel-downloads")]
         num_parallel_downloads: Option<usize>,
-        /// If true, restore from formal (slim, DB agnostic) snapshot. Note that this is only supported
-        /// for protocol versions supporting `commit_root_state_digest`. For mainnet, this is
-        /// epoch 20+, and for testnet this is epoch 12+
-        #[clap(long = "formal")]
-        formal: bool,
```

Comment:
Would it make sense to leave in `formal` for now, with a `// DEPRECATED` comment, and in the very beginning of the handler do something like 

```
if formal {
  println!("--formal flag is deprecated. Please instead use `sui-tool download-formal-snapshot`");
  return Ok(());
}
  ```
  
  Would avoid some questions and confusion from the community